yq/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-alpine
+ARG VERSION=2.9.2
+ARG JQ_VERSION=1.6
+RUN wget -O /usr/local/bin/jq https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 && \
```

Comment:
Should this be the `yq` downloads, not `jq`?

Reply:
yq requires jq underneath

Reply:
...or a required dependency perhaps?

Reply:
I was checking, `pip install jq` does not install the jq command line, so the easiest way to install the jq command line is to use the download page https://stedolan.github.io/jq/download/